src/components/firstProfile/Status.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react'
+import Logo from '../logo/Logo'
+import { Typography, Box, Button } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import { commonStyles } from 'styles/commonStyles'
+
+const STATUSES: Array<string> = [
+  'Looking for new friends',
+  "I’m learning a new language. Let's talk!",
+  "Let's be friends, I'm new in town",
+  'I look for emotional support',
+  'My plans are to move abroad',
+  "I'm a new mom. Need some help",
+  "Let's take the dogs for a walk",
+]
+
+const Status = () => {
+  const { classes } = useStyles()
+  const commonClasses = commonStyles().classes
+
+  const [selectedStatuses, setSelectedStatuses] = useState<string[]>([])
+
+  const toggleStatus = (status: string) => {
+    if (selectedStatuses.includes(status)) {
+      setSelectedStatuses(selectedStatuses.filter((s) => s !== status))
+    } else {
+      if (selectedStatuses.length < 3) {
+        setSelectedStatuses([...selectedStatuses, status])
+      } else {
+        alert('You can only select up to 3 statuses.')
+      }
+    }
+  }
+
+  const nextHandler = () => {
+    const selectedStatusesPhrases = selectedStatuses.map(
+      (index: string) => STATUSES[Number(index.slice(6))]
+    )
+    localStorage.setItem(
+      'selectedStatuses',
+      JSON.stringify(selectedStatusesPhrases)
+    )
+  }
+  return (
+    <Box className={`${commonClasses.mainBox} ${classes.mainBox}`}>
+      <Box>
+        <Logo />
+      </Box>
+      <Box className={classes.prevPageContainer}>
+        <Button
+          className={classes.prevPageButton}
+          startIcon={
+            <img
+              alt="previous page"
+              src={'/img/navigationIcons/arrow_back.svg'}
+              style={{ margin: 0 }}
+            />
+          }
+        />
+      </Box>
+      <Box className={classes.titleContainer}>
+        <Typography
+          variant="h1"
+          className={`${commonClasses.title}
+          ${classes.title}`}
+        >
+          What are you looking for?
+        </Typography>
+      </Box>
+      <Box className={classes.tipContainer}>
+        <Typography className={`${commonClasses.subTitle} ${classes.subTitle}`}>
+          This will be your status. You can always change it
+        </Typography>
+        <Typography className={classes.subTitle}>
+          Please, choose 3 statuses maximum
+        </Typography>
+      </Box>
+      <Box className={classes.phraseWrapper}>
+        {STATUSES.map((phrase, index) => (
+          <Box
+            key={phrase}
+            className={classes.phrase}
+            onClick={(event) => {
+              const targetEvent = event.target as Element
```

Comment:
could you explain why do you need this logic?

Reply:
Since phrase boxes consist of 2 elements: Box and Typography inside of it, it's essential to track user clicks on both of these elements. Since assigning the same id to both elements is not possible, we resort to using the .closest() method, which allows us to find the nearest HTML element with the specified id (containing the text "phrase").

Reply:
it's weird, i tested code and it worked without such logic

Reply:
you can remove typography and keep Box to avoid that mess with ids

Reply:
- [x] removed typography and .closest() method